website/docs/cloud-docs/projects/best-practices.mdx

```diff
@@ -0,0 +1,36 @@
+
+Projects offer several advantages to help you further develop your workspace strategy:
+
+- **Increased workspace organization**: You can add related workspaces to projects to simplify and organize a team's workspace view.
+- **Simplified management**: You can create project-level permissions and variable sets that apply to all current and future workspaces in the project. For example, you can create a project variable set containing your cloud provider credentials and all workspaces in the project will have access to them.
+- **Reduced risk with centralized control**: You can scope project permissions to only grant teams administrator access to the projects and workspaces they need.
+- **Efficiency through self-service**: Granting users project-level administrator permissions lets them create and manage workspaces.
+
+When using projects, we recommend the following:
+
+- Automate the creation of projects, variable sets, and teams together using the [TFE provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs).
+- Restrict the number of project administrators similar to maintain least privilege.
+- Designate a “landing zone” project that will contain workspaces used to create all other projects, teams, and workspaces. This lets you have a variable set that includes the organization token, which the TFE provider can use to create other resources in your organization. You can also create a [Sentinel policy](/terraform/cloud-docs/policy-enforcement) to prevent users in other projects from accessing the organization token.
+
+Finally, decide on the logical boundaries for your projects. Some considerations to keep in mind include:
+
+- **Provider boundaries**: For smaller organizations, creating one project per cloud account may make it easier to manage access. Projects can use [dynamic credentials](/terraform/tutorials/cloud/dynamic-credentials) by configuring a project variable set to avoid hard-coding long-lived static credentials.
+- **Least privilege**: You can create teams and grant them access to projects with workspaces of similar levels of criticality. For example, a production networking workspace should be in a separate project from a development compute workspace.
+- **Variable set usage**: Project-wide variable sets let you configure and reuse values such as default tags for cost-codes, owners, and support contacts.
+- **Practitioner efficiency**: Consider if it makes sense for a practitioner to need to visit multiple projects to complete a deployment.
+
+## Next Steps
+
+This article introduces some considerations to keep in mind as your organization matures their project usage. Being deliberate about how you use these to organize your infrastructure will ensure smoother and safer operations. [HCP Terraform](/terraform/tutorials/cloud-get-started) provides a place to try these concepts hands-on, and you can [get started for free](https://app.terraform.io/public/signup/account).
```

Comment:
i'd move up the workspace best practice shout out and leave the get started link for last. maybe leave out the sign up link too, feels a little marketingy and the get started tutorials will have them do it anyway.